tap_typeform/__init__.py

```diff
@@ -20,33 +20,35 @@ def validate_form_ids(client, config):
     """Validate the form ids passed in the config"""
     form_stream = Forms()
 
+    api_forms = {form.get('id') for res in form_stream.get_forms(client) for form in res if form}
```

Comment:
This should probably be within the "if not" block on 24 to make sure we're not getting all forms even when a user has provided forms.

Reply:
This is also needed to validate the form ids configured by the user, so keeping it as it is

Reply:
Agree with @kethan1122 ; this step is used to verify that the form id's provided are valid ids. It now is also used to capture all forms from the account.